I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 1 more time]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 2 more times]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 1 more time]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 2 more times]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 25, in script call
    call see_expeditions_for_region(1)
  File "game/base_of_operations.rpy", line 74, in script
    call screen expedition_screen(regionNumber)
  File "game/screens.rpy", line 3247, in execute
    screen expedition_screen(regionNumber):
  File "game/screens.rpy", line 3247, in execute
    screen expedition_screen(regionNumber):
  File "game/screens.rpy", line 3256, in execute
    frame:
  File "game/screens.rpy", line 3263, in execute
    vbox:
  File "game/screens.rpy", line 3266, in execute
    if mode == "list":
  File "game/screens.rpy", line 3329, in execute
    vbox:
  File "game/screens.rpy", line 3336, in execute
    vbox:
  File "game/screens.rpy", line 3340, in execute
    vbox:
  File "game/screens.rpy", line 3343, in execute
    text "([selected_expedition.expedition_type])" size 30 color "#000000"
AttributeError: 'BossExpedition' object has no attribute 'expedition_type'

-- Full Traceback ------------------------------------------------------------

Traceback (most recent call last):
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 1 more time]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 2 more times]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 1 more time]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 33, in script call
    call rest()
[Previous line repeated 2 more times]
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 70, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 44, in script call
    call base_travel_menu()
  File "game/places.rpy", line 33, in script call
    call rest()
  File "game/places.rpy", line 35, in script call
    call base_travel_menu()
  File "game/places.rpy", line 15, in script call
    call go_to_computer()
  File "game/base_of_operations.rpy", line 5, in script call
    call see_expeditions(1)
  File "game/base_of_operations.rpy", line 25, in script call
    call see_expeditions_for_region(1)
  File "game/base_of_operations.rpy", line 74, in script
    call screen expedition_screen(regionNumber)
  File "renpy/ast.py", line 2231, in execute
    self.call("execute")
    ~~~~~~~~~^^^^^^^^^^^
  File "renpy/ast.py", line 2195, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/statements.py", line 379, in call
    return method(parsed, *args, **kwargs)
           ~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/common/000statements.rpy", line 695, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
                    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/exports/statementexports.py", line 348, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
         ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/ui.py", line 304, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/display/core.py", line 2219, in interact
    repeat, rv = self.interact_core(
                 ~~~~~~~~~~~~~~~~~~^
        preloads=preloads,
        ^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
        **kwargs,
        ^^^^^^^^^
    )  # type: ignore
    ^                
  File "renpy/display/core.py", line 2757, in interact_core
    root_widget.visit_all(lambda d: d.per_interact())
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/displayable.py", line 432, in visit_all
    d.visit_all(callback, seen)
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "renpy/display/screen.py", line 503, in visit_all
    callback(self)
    ~~~~~~~~^^^^^^
  File "renpy/display/core.py", line 2757, in <lambda>
    root_widget.visit_all(lambda d: d.per_interact())
                                    ~~~~~~~~~~~~~~^^ 
  File "renpy/display/screen.py", line 514, in per_interact
    self.update()
    ~~~~~~~~~~~^^
  File "renpy/display/screen.py", line 715, in update
    self.screen.function(**self.scope)
    ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "game/screens.rpy", line 3247, in execute
    screen expedition_screen(regionNumber):
  File "game/screens.rpy", line 3247, in execute
    screen expedition_screen(regionNumber):
  File "game/screens.rpy", line 3256, in execute
    frame:
  File "game/screens.rpy", line 3263, in execute
    vbox:
  File "game/screens.rpy", line 3266, in execute
    if mode == "list":
  File "game/screens.rpy", line 3329, in execute
    vbox:
  File "game/screens.rpy", line 3336, in execute
    vbox:
  File "game/screens.rpy", line 3340, in execute
    vbox:
  File "game/screens.rpy", line 3343, in execute
    text "([selected_expedition.expedition_type])" size 30 color "#000000"
  File "renpy/text/text.py", line 2195, in __init__
    self.set_text(text, scope, substitute)  # type: ignore
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^                
  File "renpy/text/text.py", line 2329, in set_text
    i, did_sub = renpy.substitutions.substitute(i, scope, substitute)  # type: ignore
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^                
  File "renpy/substitutions.py", line 355, in substitute
    s = interpolate(s, variables)  # type: ignore
        ~~~~~~~~~~~^^^^^^^^^^^^^^                
  File "renpy/substitutions.py", line 89, in interpolate
    raise e
  File "renpy/substitutions.py", line 81, in interpolate
    value = renpy.python.py_eval(code, {}, scope)
            ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "renpy/python.py", line 1292, in py_eval
    return py_eval_bytecode(code, globals, locals)
           ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "renpy/python.py", line 1285, in py_eval_bytecode
    return eval(bytecode, globals, locals)
           ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<none>", line 1, in <module>
AttributeError: 'BossExpedition' object has no attribute 'expedition_type'

Windows-11-10.0.26120-SP0 AMD64
Ren'Py 8.4.1.25072401
Protectors 1.0
Tue Sep  9 09:44:35 2025
